R/pr-predict.R

```diff
@@ -70,7 +73,8 @@ vetiver_pr_post <- function(pr,
                             vetiver_model,
                             path = "/predict",
                             debug = is_interactive(),
-                            ...) {
+                            ...,
+                            check_ptype = TRUE) {
```

Comment:
This looks great! The `check_ptype` arg will have the same experience on the python side 👍